runtime-modules/council/src/benchmarking.rs

```diff
@@ -817,81 +816,81 @@ mod tests {
     fn test_announce_candidacy() {
         let config = default_genesis_config();
         build_test_externalities(config).execute_with(|| {
-            assert_ok!(test_benchmark_announce_candidacy::<Runtime>());
+            assert_ok!(Council::<Runtime>::test_benchmark_announce_candidacy());
```

Comment:
yes these fixes are on the the update-tests PR

Reply:
I've used that macro in the membership benchmarking. Its nice, but you can only use it with a single way of setting up externalities. While I was migrating other pallets I found some tests use different initial setups so just to save time I didn't bother using it everywhere, in case we later needed later to have different setups.